.eslintrc.json

```diff
@@ -12,5 +12,6 @@
         "ecmaVersion": "latest"
     },
     "rules": {
+      "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }]
```

Comment:
left-over?

Reply:
necessary for eslint. Remove it and eslint should complain about var `_`.